lib/src/Uncertainty/Algorithm/Simulation/ExpectationSimulationAlgorithm.cxx

```diff
@@ -322,7 +322,8 @@ Graph ExpectationSimulationAlgorithm::drawExpectationConvergence(const UnsignedI
       dataUpperBound.add(pt);
     }
   }
-  const Curve estimateCurve(dataEstimate, "red", "solid", 2, "expectation estimate");
+  Curve estimateCurve(dataEstimate, "expectation estimate");
```

Comment:
This code is correct, but the code below is not: `const Curve lowerBoundCurve(dataLowerBound, "green", "solid", 1, "bounds");`. This is because the `green` color is used in both the lower and upper bounds. 

Here is the result:
![image](https://github.com/openturns/openturns/assets/31351465/40dc1777-6912-47d0-a4c8-6d9f24dbceab)

In this case, the second default color in the default palette should be get and used in the code.